### DISCUSSION TITLE
Can I use this to swap species of gift/starter pokemon or scripted wild battles like Ghost Marowak or the pokemon chasing the professor in RSE? if so, how?

### DISCUSSION THREAD

**Discussion by 3DSGBA-Man** [haven1433/HexManiacAdvance#110] [Q&A]
I crawled through the wiki, and admittedly i don't have a lot of knowledge on this subject, but I want to simply swap the species of these around with something else already in the ROM, not edit in a new species in their place, and I can't quite figure out how (Example: starter Bulbasaur -> Oddish or Bellsprout, not Bulbasaur -> custom sprite pokemon like Rowlett or any sort of thing not in the base ROM).

Thank you if you take the time to respond, this program has already been so much help in editing!

**Comment by haven1433** [comment]
In the simplest terms, HMA is a data editor. That means it's pretty good at editing anything that's in a table, but has a hard time editing the weird special things that the game does as one-off code. For example, changing a moves PP or Power is easy, but changing an abilities effect is difficult since abilities are each coded individually.

The starters in firered are decided by scripts in oak's lab, so that's pretty easy to change. But the Marowak battle is hidden in some function somewhere, so there's not an easy way to edit it. Same with the zigzagoon/prof and ralts/wally encounters.

**Reply by 3DSGBA-Man** [reply]
Thanks for the reply!
